### DISCUSSION TITLE
Zigbee Green Power and broadcasts

### DISCUSSION THREAD

**Discussion by ronniebach** [Koenkk/zigbee2mqtt#20589] [General]
I experience my Zigbee network as unstable once in a while.

I have several of these Green Power PowerTag power sensors:
https://www.zigbee2mqtt.io/devices/A9MEM1570.html#schneider%2520electric-a9mem1570

I came across this:
https://www.zigbee2mqtt.io/advanced/zigbee/02_improve_network_range_and_stability.html#broadcasts
And as I can read Green Power devices might generate massive Broadcasts if not paired directly against one repeater.

From wireshark sniffing I can confirm huge amount of broadcasts from these devices.

Just as this issue mentioned.... 
https://github.com/Koenkk/zigbee2mqtt/discussions/17263

I tried to remove a device, and pair it again with one specific repeater only. 
This should according to above article cause it to Unicast instead of broadcast
However it still seems to generate lots of broadcasts !?

a) Does it take some time before alle repeaters in network "forget" that they should not proxy GP for this device?
b) What else can be done to reduce number of broadcasts from these devices?

**Comment by tausen** [comment]
> a) Does it take some time before alle repeaters in network "forget" that they should not proxy GP for this device?

As far as I recall, if you remove the GP device *without* force remove, the coordinator will ask all devices to forget it and the change should be instant. You can test it by removing the device, then pressing a key and confirming no nodes should repeat the GP message. Make sure the network is not congested when you remove and let the removal propagate through the network before pressing anything on the GP device, though (wait >10s to be sure).
